README.md

```diff
@@ -1,11 +1,27 @@
 # Amazon Chime SDK Classroom Demo
 
-This demo shows how to use the Amazon Chime SDK to build an online classroom in Electron and React.
+This demo shows how to use the Amazon Chime SDK to build an online classroom in Electron and React. This demo will ship two packages to the end user, one for windows and another one for mac.
 
 <img src="resources/readme-hero.jpg" width="640" alt="Amazon Chime SDK Classroom Demo">
 
 ## Installation
 
+To package this demo, there are two options available. The first option is to package the demo via Cloud9 and the second option is to package from the local machine.
+
+The two options are very similar in their internal working. On a high level the following steps are performed in the packaging of the demo:
+
+1. CLI arguments are parsed and the installation of prerequisites is verified. `yarn` is installed globally in the environment.
+2. Cloud9 volume size is increased (only for option 1).
```

Comment:
Can we just say for option 1, cloud9 volume size might be increase if <condition>